alethzero-latest/tools/chocolateyInstall.ps1

```diff
@@ -1,20 +1,32 @@
-$packageName    = 'AlethZero-latest'

```

Comment:
@chevdor Ether have updated their content to reflect the latest change. So this will work on the newer change and also reduces dependency of 7Zip installation :) Please test before merging to be sure.